src/main/kotlin/net/irisshaders/lilybot/extensions/moderation/TemporaryModeration.kt

```diff
@@ -104,17 +105,17 @@ class TemporaryModeration : Extension() {
 
 			check { anyGuild() }
 			check { hasPermission(Permission.ModerateMembers) }
+			check { configPresent() }
 
 			action {
-				val actionLogId = getConfigPrivateResponse("modActionLog") ?: return@action
-
+				val config = DatabaseHelper.getConfig(guild!!.id)!!
+				val actionLog = guild?.getChannel(config.modActionLog) as GuildMessageChannelBehavior
 				val userArg = arguments.userArgument
-				val actionLog = guild?.getChannel(actionLogId) as GuildMessageChannelBehavior
 
 				isBotOrModerator(userArg, "warn") ?: return@action
 
 				DatabaseHelper.setWarn(userArg.id, guild!!.id, false)
-				val newStrikes = getWarn(userArg.id, guild!!.id)
+				val newStrikes = getWarn(userArg.id, guild!!.id)?.strikes
```

Comment:
Why can we just call getWarn directly?

Reply:
We appear to be importing it manually, rather than qualifying it. Fixed in next commit